### ISSUE TITLE
Debug build cargo build -p near doesn't fully work when copy to another computer

### ISSUE THREAD

**Issue by ailisp** [nearprotocol/nearcore#1822]
`near init` and `near run` seems to work. But `near testnet` doesn't. From a recent staging build:
```
bo_nearprotocol_com@bo-pytest:~/nearcore/pytest$ RUST_BACKTRACE=1 ../target/debug/near testnet
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:1165:5
stack backtrace:
   0: backtrace::backtrace::libunwind::trace
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.37/src/backtrace/libunwind.rs:88
   1: backtrace::backtrace::trace_unsynchronized
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.37/src/backtrace/mod.rs:66
   2: std::sys_common::backtrace::_print_fmt
             at src/libstd/sys_common/backtrace.rs:76
   3: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
             at src/libstd/sys_common/backtrace.rs:60
   4: core::fmt::write
             at src/libcore/fmt/mod.rs:1028
   5: std::io::Write::write_fmt
             at src/libstd/io/mod.rs:1412
   6: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:64
   7: std::sys_common::backtrace::print
             at src/libstd/sys_common/backtrace.rs:49
   8: std::panicking::default_hook::{{closure}}
             at src/libstd/panicking.rs:196
   9: std::panicking::default_hook
             at src/libstd/panicking.rs:210
  10: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:473
  11: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:380
  12: rust_begin_unwind
             at src/libstd/panicking.rs:307
  13: core::panicking::panic_fmt
             at src/libcore/panicking.rs:84
  14: core::result::unwrap_failed
             at src/libcore/result.rs:1165
  15: core::result::Result<T,E>::unwrap
             at /rustc/032a53a06ce293571e51bbe621a5c480e8a28e95/src/libcore/result.rs:933
  16: near::config::GenesisConfig::test_with_seeds
             at near/src/config.rs:470
  17: near::config::GenesisConfig::test_sharded
             at near/src/config.rs:542
  18: near::config::create_testnet_configs_from_seeds
             at near/src/config.rs:757
  19: near::config::create_testnet_configs
             at near/src/config.rs:795
  20: near::config::init_testnet_configs
             at near/src/config.rs:813
  21: near::main
             at near/src/main.rs:116
  22: std::rt::lang_start::{{closure}}
             at /rustc/032a53a06ce293571e51bbe621a5c480e8a28e95/src/libstd/rt.rs:64
  23: std::rt::lang_start_internal::{{closure}}
             at src/libstd/rt.rs:49
  24: std::panicking::try::do_call
             at src/libstd/panicking.rs:292
  25: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:80
  26: std::panicking::try
             at src/libstd/panicking.rs:271
  27: std::panic::catch_unwind
             at src/libstd/panic.rs:394
  28: std::rt::lang_start_internal
             at src/libstd/rt.rs:48
  29: std::rt::lang_start
             at /rustc/032a53a06ce293571e51bbe621a5c480e8a28e95/src/libstd/rt.rs:64
  30: main
  31: __libc_start_main
  32: _start
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

```

**Comment by ailisp** [comment]
Problem seems to be `Genesis::test_with_seeds`. It loads `test_contract_rs.wasm` in a host specific location at run time. This is likely affected our docker images besides release binary build. A possible fix will be load the content at compile time

**Comment by ailisp** [comment]
fixed in #1826
